openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/ActivityFeed.spec.ts

```diff
@@ -0,0 +1,139 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import test, { expect } from '@playwright/test';
+import { TableClass } from '../../support/entity/TableClass';
+import { createNewPage, redirectToHomePage } from '../../utils/common';
+import { clickOnLogo } from '../../utils/sidebar';
+import { createDescriptionTask, createTagTask } from '../../utils/task';
+
+// use the admin user to login
+test.use({ storageState: 'playwright/.auth/admin.json' });
+
+const entity = new TableClass();
+
+test.describe('Activity feed', () => {
+  test.beforeAll('Setup pre-requests', async ({ browser }) => {
+    const { apiContext, afterAction } = await createNewPage(browser);
+
+    await entity.create(apiContext);
+
+    await afterAction();
+  });
+
+  test.beforeEach('Visit on landing page', async ({ page }) => {
+    await redirectToHomePage(page);
+    await entity.visitEntityPage(page);
+  });
+
+  test.afterAll('Cleanup', async ({ browser }) => {
+    const { apiContext, afterAction } = await createNewPage(browser);
+    await entity.delete(apiContext);
+    await afterAction();
+  });
+
+  test('Assigned task should appear to task tab', async ({ page }) => {
+    const value = {
+      term: entity.entity.name,
+      displayName: entity.entity.name,
+      entity: entity.type,
+      serviceName: entity.service.name,
+      entityType: entity.type,
+      assignee: 'admin',
```

Comment:
you are assigning entity.type to 2 places!
also please create a new user instead of relying on "admin"